girder/web_client/src/views/widgets/ItemListWidget.js

```diff
@@ -140,6 +151,108 @@ var ItemListWidget = View.extend({
         this.checked = _.map(this.$('.g-list-checkbox:checked'), function (checkbox) {
             return $(checkbox).attr('g-item-cid');
         }, this);
+    },
+
+    centerSelected: function () {
+        const widgetcontainer = $('.g-hierarchy-widget-container');
+        const selected = $('li.g-item-list-entry.g-selected');
```

Comment:
In general, we should use `this.$` instead of `$` whenever we're referring to elements underneath the current component, but it looks like the first one of these may be referring to a parent component. In that case, we should still be looking relative to ourselves rather than globally, so we can use `this.$el.parents('.g-hierarchy-widget-container')`.